packages/labs/cli/src/lib/lit-cli.ts

```diff
@@ -276,7 +281,8 @@ export class LitCli {
     }
     const installFrom = reference.installFrom ?? reference.importSpecifier;
     const child = childProcess.spawn(
-      'npm',
+      // https://stackoverflow.com/questions/43230346/error-spawn-npm-enoent
+      /^win/.test(process.platform) ? 'npm.cmd' : 'npm',
```

Comment:
huh! what's npm.cmd I wonder? searching around didn't find anything obvious in the npm documentation. maybe this is a weird windows PATH thing?

Reply:
I'm guessing that the way windows does the extension search for bare commands (e.g. `.bat`, `.cmd`) doesn't work via `childProcess.spawn`. 

FWIW, `childProcess.exec` doesn't have this issue; it takes a whole string with args space separated, which makes me think it does use something closer to what happens on an actual command line. I had separately used it here: https://github.com/lit/lit/blob/main/packages/labs/gen-utils/src/lib/package-utils.ts#L97. It doesn't have as nice support for streaming stdin/out though.